### ISSUE TITLE
Require Python '==2.7' 

### ISSUE THREAD

**Issue by madest92** [python-diamond/Diamond#708]
I'm having a problem installing the package:
```
pip2 install -U -e git+https://github.com/python-diamond/Diamond.git#egg=diamond
Obtaining diamond from git+https://github.com/python-diamond/Diamond.git#egg=diamond
  Updating /src/diamond clone
diamond requires Python '==2.7' but the running Python is 2.7.13
```

**Comment by NotSqrt** [comment]
It seems the line `python_requires='==2.7',` in setup.py, would be better as `~= 2.7` maybe ?

**Comment by nameprabhat** [comment]
Yes it works after changing it to ~= 2.7 in setup.py

**Comment by shortdudey123** [comment]
Closed by #718
